tests/vmi_multus_test.go

```diff
@@ -893,17 +893,22 @@ func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAdd
 		&expect.BExp{R: console.RetValue("0")},
 	}, 15)
 	Expect(err).ToNot(HaveOccurred(), "Failed to configure address %s for interface %s on VMI %s", interfaceAddress, interfaceName, vmi.Name)
+	Expect(setUpInterface(vmi, interfaceName)).To(Succeed(), "Failed to set interface %s up on VMI %s", interfaceName, vmi.Name)
+}
 
-	cmdCheck = fmt.Sprintf("ip link set %s up\n", interfaceName)
-	err = console.SafeExpectBatch(vmi, []expect.Batcher{
+func configSRIOVInterface(vmi *v1.VirtualMachineInstance, interfaceMac, interfaceAddress string) {
+	interfaceName := fmt.Sprintf("$(ip --brief l | grep %s | cut -f1 -d' ')", interfaceMac)
```

Comment:
please consider i have this PR which once the name will be added, it can use the mac in order to find the interface name
https://github.com/kubevirt/kubevirt/pull/4335

but the PRs can continue independent and once one will be merge the other will act accoding

Reply:
Thanks. I didn't know you addressed this issue already. Will take a look.

Reply:
moreover i think that its better that we dont use grep and stuff , but iterate on the scheme in order to find the interfaceName with the supplied mac, 
again up to you if to invest on it or wait my code and so
grep feels not that good imo

Reply:
Done.